sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -654,12 +654,17 @@ def apply_WriteToBigQuery(self, transform, pcoll, options):
       return self.apply_PTransform(transform, pcoll, options)
     else:
       from apache_beam.io.gcp.bigquery_tools import parse_table_schema_from_json
+      if transform.schema == beam.io.gcp.bigquery.SCHEMA_AUTODETECT \
+          or transform.schema is None:
+        schema = transform.schema
```

Comment:
Maybe here we want schema to be none always? that way we don't have to special case further down?

Reply:
actually, I've added a comment about this here: https://issues.apache.org/jira/browse/BEAM-7382

If we have autodetection while using the BigQuerySink, we should error out, as it is not supported.